README.md

```diff
@@ -38,50 +38,55 @@
 [<img src="fastlane/metadata/android/en-US/images/tenInchScreenshots/shot_11.png" width=405>](fastlane/metadata/android/en-US/images/tenInchScreenshots/shot_11.png)
 [<img src="fastlane/metadata/android/en-US/images/tenInchScreenshots/shot_12.png" width=405>](fastlane/metadata/android/en-US/images/tenInchScreenshots/shot_12.png)
 
-## Description
+### Supported Services
 
-NewPipe does not use any Google framework libraries, nor the YouTube API. Websites are only parsed to fetch required info, so this app can be used on devices without Google services installed. Also, you don't need a YouTube account to use NewPipe, which is copylefted libre software.
+NewPipe currently supports these services:
 
-### Features
+* YouTube ([website](https://www.youtube.com/)) and YouTube Music ([website](https://music.youtube.com/)) ([wiki](https://en.wikipedia.org/wiki/YouTube))
+* PeerTube ([website](https://joinpeertube.org/)) and all its instances (open the website to know what that means!) ([wiki](https://en.wikipedia.org/wiki/PeerTube))
+* Bandcamp ([website](https://bandcamp.com/)) ([wiki](https://en.wikipedia.org/wiki/Bandcamp))
+* SoundCloud ([website](https://soundcloud.com/)) ([wiki](https://en.wikipedia.org/wiki/SoundCloud))
+* media.ccc.de ([website](https://media.ccc.de/)) ([wiki](https://en.wikipedia.org/wiki/Chaos_Computer_Club))
```

Comment:
```suggestion
* [YouTube](https://www.youtube.com/) and [YouTube Music](https://music.youtube.com/) ([Wikipedia](https://en.wikipedia.org/wiki/YouTube))
* [PeerTube](https://joinpeertube.org/) and all its instances (open the website to know what that means!) ([Wikipedia](https://en.wikipedia.org/wiki/PeerTube))
* [Bandcamp](https://bandcamp.com/) ([Wikipedia](https://en.wikipedia.org/wiki/Bandcamp))
* [SoundCloud](https://soundcloud.com/) ([Wikipedia](https://en.wikipedia.org/wiki/SoundCloud))
* [media.ccc.de](https://media.ccc.de/) ([Wikipedia](https://en.wikipedia.org/wiki/Chaos_Computer_Club))
```

Reply:
That's what I initially did, but I didn't want anyone even slightly unawares to accidentally open a website they didn't want to, e.g. YT. So I made the links explicitly clear.

Reply:
```suggestion
<!-- We link to the service websites separately to avoid people accidentally opening a website they didn't want to. -->
* YouTube ([website](https://www.youtube.com/)) and YouTube Music ([website](https://music.youtube.com/)) ([wiki](https://en.wikipedia.org/wiki/YouTube))
* PeerTube ([website](https://joinpeertube.org/)) and all its instances (open the website to know what that means!) ([wiki](https://en.wikipedia.org/wiki/PeerTube))
* Bandcamp ([website](https://bandcamp.com/)) ([wiki](https://en.wikipedia.org/wiki/Bandcamp))
* SoundCloud ([website](https://soundcloud.com/)) ([wiki](https://en.wikipedia.org/wiki/SoundCloud))
* media.ccc.de ([website](https://media.ccc.de/)) ([wiki](https://en.wikipedia.org/wiki/Chaos_Computer_Club))
```